install/updates/froxlor/0.9/update_0.9.inc.php

```diff
@@ -3893,6 +3893,12 @@
 	updateToVersion('0.9.39.2');
 }
 
+if (isFroxlorVersion('0.9.39.2-dev1')) {
+
+        showUpdateStep("Updating from 0.9.39.2 to 0.9.39.1-dev1", false);
+        updateToVersion('0.9.39.');
```

Comment:
this would be fun for future updates....you are missing "1-dev1" here. But that's wrong anyway as stated already